api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +9,12 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+    
+def query_bouquet(bouquet_id):
+    bouquet = BouquetsModel.query.filter_by(id=bouquet_id).first()
+    found_bouquet = False
+    if bouquet:
+        found_bouquet = BouquetsSchema(many=False).dump(bouquet)
```

Comment:
This variable name is not ideal. What kind of data does `found_bouquet` hold? Is it boolean or the bouquet object? Please change this variable name.